codegenerator/cli/npm/envio/src/LazyLoader.res

```diff
@@ -99,37 +99,37 @@ let rec loadNext = async (am: asyncMap<'a>, k: int) => {
     switch await awaitTaskPromiseAndLoadNextWithTimeout() {
     | _ => Promise.resolve()
     | exception err =>
-      Logging.error({
-        "err": err,
-        "msg": `EE1100: Top level promise timeout reached. Please review other errors or warnings in the code. This function will retry in ${(am._retryDelayMillis / 1000)
-            ->Belt.Int.toString} seconds. It is highly likely that your indexer isn't syncing on one or more chains currently. Also take a look at the "suggestedFix" in the metadata of this command`,
-        "metadata": am.metadata,
-      })
```

Comment:
Had to move the dependency on Logging, to move the file to npm package and be able normally edit it with ReScript watch mode enabled.